src/mango_native_proc.erl

```diff
@@ -148,8 +149,9 @@ get_text_entries({IdxProps}, Doc) ->
 
 get_text_entries0(IdxProps, Doc) ->
     DefaultEnabled = get_default_enabled(IdxProps),
+    IndexArrayLengths = get_index_array_lengths(IdxProps),
     FieldsList = get_text_field_list(IdxProps),
-    TAcc = #tacc{fields = FieldsList},
+    TAcc = #tacc{index_array_lengths = IndexArrayLengths, fields = FieldsList},
```

Comment:
This is rather long. You should reformat this to look like:

    TAcc = #tacc{
        index_arrray_lengths = IndexArrayLengths,
        fields = FieldsList
    },